package.json

```diff
@@ -30,14 +31,14 @@
     "json-server": "^0.17.4",
     "jsorm": "^1.3.22",
     "mathjs": "^12.4.1",
-    "vite": "^5.2.11",
+    "postcss": "^8.4.37",
+    "tailwindcss": "3.3.5",
+    "vite": "^5.1.6",
     "vue": "^3.4.21",
-    "vue-router": "^4.3.0",
-    "tailwindcss": "3.4.3",
-    "postcss": "^8.4.38",
-    "@sanger/ui-styling": "^4.1.0"
+    "vue-router": "^4.3.0"
```

Comment:
What caused these package removals?

Reply:
To be fair, I was going to applaud you on the reordering!  It might be nice to do a quick sort on the package names if possible.

Reply:
Oh wait!  They are already sorted 🤔